api/src/main/java/io/minio/MinioClient.java

```diff
@@ -3908,8 +3908,7 @@ private String getLatestIncompleteUploadId(String bucketName, String objectName)
    * @return an iterator of Upload.
    * @see #listIncompleteUploads(String, String, boolean)
    */
-  public Iterable<Result<Upload>> listIncompleteUploads(String bucketName, String prefix)
-    throws XmlPullParserException {
+  public Iterable<Result<Upload>> listIncompleteUploads(String bucketName, String prefix) {
```

Comment:
What is the problem of keeping `throws XmlPullParserException`?

Reply:
`throws XmlPullParserException` was never used so i have removed it.

Reply:
What do you mean by never used?  Do you say the exception never be raised?

Reply:
I have added it back.

Reply:
i have added it back